operator/internal/validation/openshift/common.go

```diff
@@ -64,23 +65,44 @@ func validateRuleExpression(namespace, tenantID, rawExpr string) error {
 	}
 
 	matchers := selector.Matchers()
-	if tenantID != tenantAudit && !validateIncludesNamespace(namespace, matchers) {
-		return lokiv1.ErrRuleMustMatchNamespace
+	if tenantID != tenantAudit {
```

Comment:
I remember asking myself why we did this validation for `infrastructure` as well. There's also a bug ([LOG-6862](https://issues.redhat.com/browse/LOG-6862)) filed for this.

Should we change the validation, so that the namespace checks only apply to `application`?

Reply:
We decided to handle this one on a follow up